book/08-customizing-git/sections/config.asc

```diff
@@ -105,33 +105,32 @@ what happened
 ".git/COMMIT_EDITMSG" 14L, 297C
 ----
 
-If your team has a commit-message policy, then putting a template for that policy on your system and configuring Git to use it by default can help increase the chance of that policy being followed regularly.
+Jika tim Anda mempunyai aturan untuk pesan-commit, maka memasang templat untuk aturan tersebut pada sistem Anda dan mengkonfigurasi Git untuk menggunakannya secara default akan membantu untuk meningkatkan kemungkinan aturan tersebut diikuti secara teratur.
 
 ===== `core.pager`
 
 (((pager)))
-This setting determines which pager is used when Git pages output such as `log` and `diff`.
-You can set it to `more` or to your favorite pager (by default, it's `less`), or you can turn it off by setting it to a blank string:
+Pengaturan ini menentukan pager yang mana yang digunakan ketika Git melakukan pages output seperti `log` dan `diff`. Anda bisa mengaturnya menjadi `more` atau ke pager favorit Anda (secara default, hal tersebut merupakan `less`), atau Anda bisa mematiknnya dengan mangaturnya ke string kosong:
 
 [source,console]
 ----
 $ git config --global core.pager ''
 ----
 
-If you run that, Git will page the entire output of all commands, no matter how long they are.
+Jika anda menjalankana perintah tersebut, Git akan mengepage semua ouput dari semua perintah, tidak perduli seberapa panjang perintah tersebut.
```

Comment:
will page the entire output of all commands maksudnya si Git itu akan menampilkan semua pesan tanpa perantara more atau less.